source/installation/cluster-config-schema.rst

```diff
@@ -190,3 +190,28 @@ See :ref:`Disable Host Link in Batch Connect Session Card <disable-host-link-bat
 .. note::
 
   The user is responsible for providing the `%s` that is used to place the script content. If a `script_wrapper` is provided without `%s` then batch connect applications are unlikely to work properly.
+
+
+******************
+Login Cluster Only
+******************
```

Comment:
This renders fine, but I feel like it's weird to have the `****` line on the top  - do headers in other files do this?

Reply:
I'm not sure what the comment means with "`****` line on the top". I'm just trying to add the subtopic under cluster, and that was the only way I could find that matched what was in the file and others to make this visible in the nav, while also keeping this in a somewhat rational place.

Were you thinking a separate page like the resource-providers and a navigation link at that depth instead of under cluster config?

Reply:
I mean the line of **** on 195 above the heading. 

This I've seen a lot throughout this repo.
```
Login Cluster Only
******************
```

This - with a line of **** above the heading - I don't think I've seen in this repo.
```
******************
Login Cluster Only
******************
```

Reply:
Yeah, we use that all over. I think you are thinking of `====` and some others. You get an error without that top line.

If you look in the `cluster-config-schema.rst` for instance, you can see it in `A Break Down`.

I like it because it seemed to give the best navigation output for a user to see this is all, but I am open to changing it.

Reply:
Huh, but yeah when I look in `kubernetes.rst` I only see the version with them on the bottom. Well then, I am happy to remove the top line, less text the better.

Reply:
Yeah I don't know, it's a build error around the title being at an inconsistent level, but when you add that top line back things work. 

This is a small thing to me that it needs that though, and I see it used elsewhere so it doesn't bother me.

Reply:
Haha, so I think the reason this file needs that is because the rest of the file was written that way.

When I remove the top lines on the other headings for `First an Example` and `A Break Down` and have only bottom lines for them all with `****` it will render normal and show correctly in the nav. So i'll include a commit to remove those other top pieces as well.